src/backend/columnar/cstore_customscan.c

```diff
@@ -214,6 +215,13 @@ CreateColumnarScanPath(RelOptInfo *rel, RangeTblEntry *rte)
 	path->parent = rel;
 	path->pathtarget = rel->reltarget;
 
+	/*
+	 * We need to set param_info for lateral join parameters.
+	 * See https://github.com/citusdata/citus/issues/4600.
+	 */
```

Comment:
Can we remove this comment in its entirety?

The comment does not explain why we need to set it, neither does the linked issue, they only show cases how it went wrong.

Instead, having the description of this PR as the commit message of this line can provide a lot of context via git blame/annotations in the IDE without paying the cost of the vertical space of a comment.

We can collapse the assignment of `path->param_info` into the block above where we set relevant fields.